pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -459,8 +459,10 @@ def select_precision_plugin(self) -> PrecisionPlugin:
                         "You have asked for native AMP on CPU, but AMP is only available on GPU."
                     )
                 if not _NATIVE_AMP_AVAILABLE:
-                    msg = "You have asked for native AMP but your PyTorch version does not support it." \
-                                          " Consider upgrading with `pip install torch>=1.6`."
+                    msg = (
+                        "You have asked for native AMP but your PyTorch version does not support it."
```

Comment:
We could mention their PyTorch version here?

Reply:
Just fixing a PEP8 complain by pre-commit here. Don't mean to change anything related to the accelerator connector